packages/@ama-sdk/schematics/schematics/index.it.spec.ts

```diff
@@ -72,6 +72,8 @@ function setupYarn(yarnVersion: string) {
 
 /**
  * Setup a new sdk using schematics CLI
+ *
+ * @deprecated A new SDK should be generated via the `npm create` feature
```

Comment:
you are in the IT test why not using the `npm create` in this one ?

Reply:
Because this is not the `@ama-sdk/create` package and using it here will create a circular dep.
The IT of the `@ama-sdk/create` will be created in the dedicated package but this function is actually deprecated and need to be removed (which will lead to the rewrite of these ITs which is not the purpose of this PR)